plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/CatalogType.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+public enum CatalogType
+{
+    HIVE,
+    // TODO: dummy type to pass IcebergConfig test, remove it after adding actual catalog types
```

Comment:
maybe we should not add config already, given we have only one implementation. And only add config when we have one more. Then you do not need this entry at all.

Reply:
The reason I added it here is to allow different developers to write implementations in parallel after this PR is merged. Otherwise this will force everyone to add that catalog type class. I think this is fine as long as we know this will be removed very soon and exception is thrown when people try to set the type to UNKNOWN.

Reply:
fair